charts/latest/blob-csi-driver/values.yaml

```diff
@@ -147,8 +147,6 @@ node:
   nodeSelector: {}
   tolerations:
     - operator: "Exists"
-  livenessProbe:
```

Comment:
why remove such port, it's used here:
https://github.com/kubernetes-sigs/blob-csi-driver/blob/2832df4ef82364d967c09d84790690912c573dfd/charts/latest/blob-csi-driver/templates/csi-blob-node.yaml#L129-L132

Reply:
VSCode hints that it duplicates with https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/charts/latest/blob-csi-driver/values.yaml#L114-L115

Reply:
yes, that's correct